acquia_lift.batch.inc

```diff
@@ -10,11 +10,12 @@
  * Batch syncs all campaigns to Lift.
  */
 function acquia_lift_batch_sync_campaigns($agents = array()) {
-  // First delete the queue because we are about to sync everything now.
-  acquia_lift_delete_queue();
-  // Now get all sync operations for all existing Acquia Lift agents.
+  // Now get all sync operations for all existing Acquia Lift testing agents.
   if (empty($agents)) {
     foreach (acquia_lift_get_agent_types() as $type => $info) {
+      if ($type == 'acquia_lift_target') {
```

Comment:
I was actually going to use that and then saw it takes an actual agent, not a type as its param. I think once simple a/b is gone we can simplify all these checks anyway.